docs/en/observability/slo-create.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[slo-create]]
+= Create an SLO 
+
+beta::[]
+
+To create and SLO, go to *Observability → SLOs*: 
+* If you're creating your first SLO, you'll see an introductory page. Click the *Create SLO* button.
+* If you have previously created SLOs, click the *Create new SLO* button in the upper-right corner of the SLO page.
```

Comment:
Should we add a section about the user's role privileges required to access this? Details about role setup for write and read access to the SLO feature: https://github.com/elastic/kibana/pull/150554#issue-1576156701